ansible_ai_connect_chatbot/src/useChatbot/useChatbot.ts

```diff
@@ -53,6 +59,15 @@ export const inDebugMode = () => {
   return import.meta.env.PROD ? debug === "true" : debug !== "false";
 };
 
+export const isStreamingSupported = () => {
+  // For making streaming mode debug easier.
+  if (!import.meta.env.PROD && import.meta.env.MODE.includes("stream")) {
```

Comment:
This is for debugging.  When the vite server is started with the `stream` mode, the UI stream support is enabled.